src/migrations/20231219100412-add-course-tables.js

```diff
@@ -0,0 +1,163 @@
+const {
+  prepMigration,
+  removeTables,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      await queryInterface.createTable('Courses', {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER,
+        },
+        name: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        nameLookUp: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
```

Comment:
What is this do/for?

Reply:
There might be a way to do this inband to the comparison where its used so that we dont have to store the striped value

Reply:
I think this is probably a better solution then nesting all sorts of sql stuff.